res/values/strings.xml

```diff
@@ -638,30 +637,30 @@
-    <string name="systemmsg_failed_sending_to">Failed to send message to %1$s.</string>
-
-    <string name="systemmsg_ephemeral_timer_disabled">Disappearing messages timer disabled.</string>
-    <string name="systemmsg_ephemeral_timer_enabled">Disappearing messages timer set to %1$s s.</string>
-    <string name="systemmsg_ephemeral_timer_minute">Disappearing messages timer set to 1 minute.</string>
-    <string name="systemmsg_ephemeral_timer_hour">Disappearing messages timer set to 1 hour.</string>
-    <string name="systemmsg_ephemeral_timer_day">Disappearing messages timer set to 1 day.</string>
-    <string name="systemmsg_ephemeral_timer_week">Disappearing messages timer set to 1 week.</string>
-    <string name="systemmsg_ephemeral_timer_four_weeks">Disappearing messages timer set to 4 weeks.</string>
-    <string name="systemmsg_ephemeral_timer_minutes">Disappearing messages timer set to %1$s minutes.</string>
-    <string name="systemmsg_ephemeral_timer_hours">Disappearing messages timer set to %1$s hours.</string>
-    <string name="systemmsg_ephemeral_timer_days">Disappearing messages timer set to %1$s days.</string>
-    <string name="systemmsg_ephemeral_timer_weeks">Disappearing messages timer set to %1$s weeks.</string>
+    <string name="systemmsg_failed_sending_to">Could not send message to %1$s.</string>
+
+    <string name="systemmsg_ephemeral_timer_disabled">Messages won't dissapear.</string>
+    <string name="systemmsg_ephemeral_timer_enabled">Messages now dissapear in %1$s seconds.</string>
+    <string name="systemmsg_ephemeral_timer_minute">Messages now dissapear after a minute.</string>
+    <string name="systemmsg_ephemeral_timer_hour">Messages now dissapear after one hour.</string>
+    <string name="systemmsg_ephemeral_timer_day">Messages now dissapear in one day.</string>
+    <string name="systemmsg_ephemeral_timer_week">Messages now dissapear in one week.</string>
+    <string name="systemmsg_ephemeral_timer_four_weeks">Messages now dissapear in 4 weeks.</string>
+    <string name="systemmsg_ephemeral_timer_minutes">Messages now dissapear in %1$s minutes.</string>
+    <string name="systemmsg_ephemeral_timer_hours">Messages now dissapear in %1$s hours.</string>
+    <string name="systemmsg_ephemeral_timer_days">Messages now dissapear in %1$s days.</string>
+    <string name="systemmsg_ephemeral_timer_weeks">Messages now dissapear in %1$s weeks.</string>
 
     <!-- Translators: Protection disabled/enabled messages may be displayed together with a "big shield" and may be suffixed by systemmsg_action_by_user or systemmsg_action_by_me -->
-    <string name="systemmsg_chat_protection_disabled">Chat protection disabled.</string>
-    <string name="systemmsg_chat_protection_enabled">Chat protection enabled.</string>
+    <string name="systemmsg_chat_protection_disabled">Chat protection off.</string>
+    <string name="systemmsg_chat_protection_enabled">Chat protection on.</string>
```

Comment:
i do not think "off" is the same as "disabled", the one is the state, the other, an action just taken by someone.

but anyway, the string might be expected by the user taking the action (see comment), so "off" would not work anyway here (we should get rid of this expansion, however, currently it is like that)

Reply:
Not necessarily, and depends on the translation.
I think there is no distinction to be made for something that can be on and disabled at that.
This is just "off" or "on" (?).